python/snewpy/models/ccsn.py

```diff
@@ -463,6 +463,29 @@ def __init__(self, axion_mass:u.Quantity, axion_coupling:u.Quantity):
         self.metadata['PNS mass'] = pns_mass*u.Msun
         return super().__init__(filename, self.metadata)
 
+@RegistryModel(
+    Bfield = ['hydro','l1_b12','l2_b12_dipdecay'],
+    direction = ['average','equator','north','south'],
+    grav=['gravA','gravB',''],
+    rotation=['','0deg','90deg'],
```

Comment:
I think it’d be slightly nicer to have
```suggestion
    grav=['A', 'B', None],
    rotation=[0, 90, None],
```
(and adjust the filename f-strings below accordingly).